package.json

```diff
@@ -48,7 +50,7 @@
     "webpack-cli": "^3.3.0"
   },
   "dependencies": {
-    "ua-parser-js": "^0.7.19"
+    "ua-parser-js": "0.7.22"
```

Comment:
@robertdumitrescu INFO - I'll assume this is the latest version?  We are aware of `null` values so I'd be interested to understand if this will offer us any improvements.

Reply:
@robertdumitrescu INFO And why do we drop compatibility caret (^) ?

Reply:
OK - done

Reply:
OK